app/services/section/remove_service.rb

```diff
@@ -9,18 +9,29 @@ def initialize(user:, manual_id:, section_uuid:, attributes:)
   end
 
   def call
+    begin
+      manual = Manual.find(manual_id, user)
+    rescue KeyError
+      raise ManualNotFoundError.new(manual_id)
```

Comment:
Ah, that's useful to know! I'm going to leave this for now as I can imagine addressing it as I continue to work through these services ensuring that the save to the publishing api happens before the save to our database.